dist/10-10EZ-schema.json

```diff
@@ -361,52 +362,8 @@
       "maxLength": 20
     },
     "stateOfBirth": {
+      "type": "string",
```

Comment:
Should we go through and fix all of these? I found one (vaMedicalFacility) for the pages I'm working on. It's consistently on fields with `enum`.